data/json/items/comestibles/mutagen.json

```diff
@@ -36,6 +54,32 @@
         "flags" : [ "MUTAGEN_CATTLE" ]
     },
     {
+        "id" : "mutagen_cephalopod",
+        "copy-from" : "mutagen",
+        "type" : "COMESTIBLE",
+        "name" : "cephalopod mutagen",
+        "addiction_potential" : 2,
+        "flags" : [ "MUTAGEN_CEPHALOPOD" ]
+    },
+    {
+        "id" : "mutagen_chimera",
+        "copy-from" : "mutagen",
+        "type" : "COMESTIBLE",
+        "name" : "chimera mutagen",
+        "description" : "An extremely rare mutagen cocktail.",
+        "addiction_potential" : 2,
+        "flags" : [ "MUTAGEN_CHIMERA" ]
+    },
+    {
+        "id" : "mutagen_elfa",
```

Comment:
Currently, you reduced price of this mutagen (and some other) from ```200 000``` to the baseline value (```100 000```) silently, that's not good.